.github/ISSUE_TEMPLATE.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Create a bug report
+labels: ["defect"]
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: "Please provide a description and a way to reproduce the problem."
+      placeholder: |
+        Please use the Yii Chat channel or IRC channel for questions.
+        
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Yii Version
+      description: Which version of Yii are you using?
+      placeholder: "2.0.49 (latest)"
+      multiple: false
+      options:
+      - 2.0.49 (latest)
+      - 2.0.48
+      - 2.0.47
+      - 2.0.46
+      - 2.0.45
+      - 2.0.44
+      - 2.0.43
+      - 2.0.42
+      - 2.0.41
+      - 2.0.40
+      - 2.0.39
+      - 2.0.38
+      - 2.0.37
+      - 2.0.36
+      - 2.0.35
+      - 2.0.34
+      - 2.0.33
+      - 2.0.32
+      - 2.0.31
+      - 2.0.30
+      - 2.0.29
+      - 2.0.28
+      - 2.0.27
+      - 2.0.26
+      - 2.0.25
+      - 2.0.24
+      - 2.0.23
+      - 2.0.22
+      - 2.0.21
+      - 2.0.20
+      - 2.0.19
+      - 2.0.18
+      - 2.0.17
+      - 2.0.16
+      - 2.0.15
+      - 2.0.14
+      - 2.0.13
+      - 2.0.12
+      - 2.0.11
+      - 2.0.10
+      - 2.0.9
+      - 2.0.8
+      - 2.0.7
+      - 2.0.6
+      - 2.0.5
+      - 2.0.4
+      - 2.0.3
+      - 2.0.2
+      - 2.0.1
+      - 2.0.0
+      - 1 
```

Comment:
That's not true for any Yii3 package. No such versions. It should be a plain text input.

Reply:
Changes were made here https://github.com/yiisoft/package-template/pull/113
I will close this PR